storage/innobase/btr/btr0cur.cc

```diff
@@ -413,6 +407,13 @@ btr_cur_latch_leaves(
 
 		latch_leaves.blocks[1] = get_block;
 #ifdef UNIV_BTR_DEBUG
+		/* Sanity check only after both the blocks are latched. */
+		if (latch_leaves.blocks[0] != NULL) {
+			ut_a(page_is_comp(latch_leaves.blocks[0]->frame)
+			     == page_get_page_no(page));
```

Comment:
This is handled by @kamil-holubicki via another pull requests #3293 and #3294 

This assertion is wrong btw,  page_is_comp()  == page_is_comp (instead of page_get_page_no())